zubhub_backend/zubhub/templates/activitylog/clap.html

```diff
@@ -0,0 +1 @@
+<strong>{{ source }}</strong> clapped for <strong>{{target}}</strong>'s project "{{ project }}"
```

Comment:
Knitpick but maybe this could be "clapped on".

Reply:
Done.